src/ui/components/set_password.c

```diff
@@ -193,7 +193,7 @@ static void _on_event(const event_t* event, component_t* component)
     case EVENT_TOGGLE_ALPHANUMERIC:
         data->auto_forward_active = true;
         data->auto_forward_counter = 0;
-        data->input_mode = (data->input_mode + 1) % NUM_INPUT_TYPES;
+        data->input_mode = (data->input_mode + 1) % 3;
```

Comment:
(See above) `NUM_INPUT_TYPES` is quite useful as a sentinel value for computing the size of the enum automatically. If we want to remove it from the enum let's define it on a side so not to leave the `3` hardcoded...

Reply:
As the commit message says, set_password.c is dead code and will be removed soon (it is outdated anyway in various ways).

Other than here, this sentinel value is not used anywhere currently.

Happy to remove set_password.c today.

Reply:
Ok, let's ditch all of it together then?

Reply:
ACK, done